core/tests/test_utils.py

```diff
@@ -2282,6 +2280,118 @@ def assert_same_list_elements(self, phrases, stdout):
 GenericTestBase = AppEngineTestBase
 
 
+class GenericEmailTestBase(GenericTestBase):
+    """Base class for tests requiring email services."""
+
+    class EmailMessageMock(python_utils.OBJECT):
+        """Mock for core.platform.models email services messages."""
+
+        def __init__(
+                self, sender_email, recipient_email, subject, plaintext_body,
+                html_body, bcc=None, reply_to=None, recipient_variables=None):
+            self.sender = sender_email
```

Comment:
Write either a docstring or a one-sentence comment above each attribute to explain fully what it is.

Probably a docstring is better since you can include types as well.